Client/Frontend/Menu/BookmarksViewController.swift

```diff
@@ -580,8 +582,14 @@ extension BookmarksViewController: NSFetchedResultsControllerDelegate {
     switch type {
     case .update:
         let update = { (path: IndexPath?) in
-            if let path = path, let cell = self.tableView.cellForRow(at: path) {
-                self.configureCell(cell, atIndexPath: path)
+            if let path = path, let cell = self.tableView.cellForRow(at: path),
+                let fetchedObjectsCount = self.bookmarksFRC?.fetchedObjects?.count {
```

Comment:
I can place the out of bounds checker either here or in `configureCell()`.
Choose to place it here in case we want to redirect this update to `tableView.delete()` for example(not needed for now)

Reply:
I think here is good, I would also throw a `fatalError` inside of `configureCell` honestly. That should never be called, and should just fail aggressively IMO.